estuary_updater/handlers/freshmaker.py

```diff
@@ -89,16 +89,20 @@ def build_state_handler(self, msg):
         :param dict msg: a message to be processed
         """
         build = msg['body']['msg']
-        self.create_or_update_build(build)
+        self.create_or_update_build(build, msg)
 
-    def create_or_update_build(self, build):
+    def create_or_update_build(self, build, msg):
```

Comment:
You could consider making the message parameter be just the freshmaker event ID.